routes/api/users.js

```diff
@@ -61,4 +61,18 @@ router.put("/:username",(request,response) =>{
      response.send({msg:"User deleted",data:newdata});
 })
 
+router.put("/:username/grades",(request,response) =>{
+    const x=request.body;
```

Comment:
rename x to be understandable by the reader.
Like `const body = request.body` or another better option,
`const courseName = request.body.courseName` and make the other constants for `grade` and `creditHours`.